server/container/guice/memory-guice/src/main/java/org/apache/james/MemoryJamesServerMain.java

```diff
@@ -73,6 +74,7 @@
         new MailboxRoutesModule(),
         new MailQueueRoutesModule(),
         new MailRepositoriesRoutesModule(),
+        new MessagesRoutesModule(),
```

Comment:
Shouldn't it me done in the previous commit?

Reply:
I might have mixed up my squashing before pushing, sorry